gke/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: gde
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: gde
+    networking.gke.io/managed-certificates: gde
+    kubernetes.io/ingress.class: "gce"
+spec:
+  rules:
+  - http:
+      paths:
+      - path: /*
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: gde
+            port:
+              number: 80
```

Comment:
Could you briefly comment on this port?

Reply:
this is which the client tries to hit. `80` is the default port number for HTTP connection I guess. But It could be modified to any number. 

`service.name` is what `Ingress` is trying to find the `Service` entity.